package.json

```diff
@@ -4,34 +4,32 @@
 	"description": "Creates a Markdown table summarizing your GitHub sponsors. 💰",
 	"repository": {
 		"type": "git",
-		"url": "https://github.com/JoshuaKGoldberg/github-sponsors-to-markdown"
+		"url": "git+https://github.com/JoshuaKGoldberg/github-sponsors-to-markdown.git"
 	},
 	"license": "MIT",
 	"author": {
 		"name": "JoshuaKGoldberg",
 		"email": "npm@joshuakgoldberg.com"
 	},
 	"type": "module",
-	"main": "./lib/index.js",
+	"main": "lib/index.js",
 	"bin": "./bin/github-sponsors-to-markdown.js",
```

Comment:
🐛 the bin should be simplified to remove a preceding `./`. Keeping the `./` in there doesn't work for other tools: e.g. `blockESLint`'s `allowDefaultProject`.

https://github.com/JoshuaKGoldberg/create-typescript-app/issues/2081